packages/toolpad-app/src/toolpadDataSources/rest/server.ts

```diff
@@ -133,10 +131,16 @@ async function execBase(
   fetchQuery: FetchQuery,
   params: Record<string, string>,
 ): Promise<FetchResult> {
+  const queryScope = {
+    // TODO: remove deprecated query after v1
+    query: params,
```

Comment:
Do we need to write a migration to deprecate this?

Reply:
It's require us to rewrite javascript expressions and replace certain global variables. It's doable, but it'd require us to pull in quite some heavy dependencies. We can deal with it when a more pressing need arises